plusdeck/client.py

```diff
@@ -209,11 +327,17 @@ def _on_state(self, state: State):
 
         if previous == State.UNSUBSCRIBING:
             if not (state == State.PAUSED_A or state == State.PAUSED_B):
-                raise SubscriptionError(f"Unexpected state {state} while unsubscribing")
```

Comment:
I should be able to continue raising these errors, as they're handled in `data_received`.